.bumpversion.cfg

```diff
@@ -1,9 +1,6 @@
 [bumpversion]
 current_version = 0.1.3
 commit = True
-tag = True
-
-[bumpversion:file:setup.py]
 
```

Comment:
doc/conf.py?

Reply:
What about it? It's below, github just decided to hide it and you need to expand

Reply:
yes, sorry